Changelog.md

```diff
@@ -401,7 +401,7 @@ Bugfixes:
 
 Features:
  * C API (``jsonCompiler``): Export the ``license`` method.
- * Code Generator: Optimise the fallback function, by removing a useless jump.
+ * Code Generator: Optimise the fallback function, by removing an useless jump.
```

Comment:
I'm pretty sure this was correct. Please check again.

Reply:
yeah, it is correct... I will change them back